reports/github.go

```diff
@@ -0,0 +1,52 @@
+package reports
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	ghinstallation "github.com/bradleyfalzon/ghinstallation/v2"
+	"github.com/google/go-github/v57/github"
+	"golang.org/x/exp/slog"
+)
+
+var (
+	gh              *github.Client
+	ghTransport     *ghinstallation.Transport
+	ghAppName       = os.Getenv("CICD_ROBOT_GITHUB_APP_NAME")
+	ghUserName      = fmt.Sprintf("%s[bot]", ghAppName)
+	ghAppPrivateKey = os.Getenv("CICD_ROBOT_GITHUB_APP_PRIVATE_KEY")
+)
+
+func init() {
+	ghAppID, err := strconv.ParseInt(os.Getenv("CICD_ROBOT_GITHUB_APP_ID"), 10, 32)
```

Comment:
would separate out the getEnv to a var as you have above, or does the init order make that impossible?

Reply:
separating it would would probably make this more readable. I will do that.